psalm-baseline.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If you manage to fix these two errors, we can avoid a baseline file in this project which would be awesome. 👍🏼

Reply:
Maybe casting a non-native typehinted "string" argument to string might help to tell psalm whats going on 👍🏼

Reply:
The one in the test can be fixed by removing that test/merging it into the dependent test. Does that seem reasonable?

The one in `FlashMessages` is trickier. It's not about not recognizing a string. Rather, psalm really doesn't like `mixed`, but the public API here explicitly accepts `mixed $value`. 

With the caveat that I'm pretty new to psalm, it seems like psalm considers `mixed` to be a kind of abomination to be exorcised, and not just a maximally-wide type (like `any` in typescript).  In this instance, psalm knows full well (according to `@psalm-trace` if you you add some) that we're assigning `mixed` to `mixed`. But it appears psalm seems to take a hard line that you shouldn't do that. See: https://psalm.dev/r/69768c9c2a

It's equally (if not more) possible that I'm missing something, and I share your dream of not needing a baseline, so I'm going to give others with more psalm-experience a change to speak up.

Reply:
Per our brief discussion in slack, I've suppressed this error, as it's unavoidable.